test/unit/test_disk_tensor.py

```diff
@@ -302,5 +302,43 @@ def test_bf16_disk_write_read(self):
     ct = t.llvm_bf16_cast(dtypes.float)
     assert ct.numpy().tolist() == [9984., -1, -1000, -9984, 20]
 
+  @unittest.skipUnless(Device.DEFAULT in {"NV", "AMD"}, "only AMD and NV supports ")
```

Comment:
These tests should work everywhere, right? Aside from checking if io_uring is there it's just a normal copy.